doc/modules/cassandra/pages/auto-repair/overview.adoc

```diff
@@ -0,0 +1,125 @@
+= Overview of Auto Repair
+:navtitle: Auto Repair overview
+:description: Auto Repair concepts - How it works, how to configure it, and more.
+:keywords: CEP-37
+
+Repair orchestration inside Apache Cassandra. This fully Automated Repair scheduler inside Cassandra does not depend on the control plane, significantly reducing our operational overhead.
+At a high level, a dedicated thread pool is assigned to the repair scheduler. The repair scheduler inside Cassandra maintains a new replicated table under a distributed system_distributed keyspace. This table maintains the repair history for all the nodes, such as when it was repaired the last time, etc. The scheduler will pick the node(s) that run the repair first and continue orchestration to ensure Every table and all of their token ranges are repaired. The algorithm can also run repairs simultaneously on multiple nodes and splits the token range into subranges with the necessary retry to handle transient failures. The automatic repair runs as soon as we start a Cassandra cluster, like Compaction, and does not require human-in-the-loop.
```

Comment:
a new replicated table under a distributed system_distributed keyspace.

to:

a new replicated table, system_distributed.auto_repair_history.

Reply:
remove ', etc.'.

Reply:
Every table -> every table.

Compaction -> compaction.

Reply:
Done